Dockerfiles/nanoserver/Dockerfile

```diff
@@ -1,15 +1,12 @@
-FROM microsoft/dotnet:2.1-runtime-nanoserver-sac2016
+FROM mcr.microsoft.com/dotnet/runtime:3.1-nanoserver-20H2
 ARG OCTO_TOOLS_VERSION=4.31.1
 
-LABEL maintainer="devops@octopus.com" 
+LABEL maintainer="devops@octopus.com"
 LABEL octopus.dockerfile.version="1.0"
-LABEL octopus.tools.version=$OCTO_TOOLS_VERSION 
+LABEL octopus.tools.version=$OCTO_TOOLS_VERSION
 
-COPY OctopusTools.$OCTO_TOOLS_VERSION.portable.zip ./octo/OctopusTools.zip
-
-RUN Expand-Archive ./octo/OctopusTools.zip -DestinationPath octo; \
-	Remove-Item -Force ./octo/OctopusTools.zip; \
-	mkdir src |Out-Null
+ADD OctopusTools.$OCTO_TOOLS_VERSION.portable.tar.gz ./octo/
```

Comment:
We're using the linux tar.gz... It claims it's `portable`, so while weird, it seems valid...

Comparing the `.portable.zip` and `.portable.tar.gz` files says they're identical:
![image](https://user-images.githubusercontent.com/373389/121977252-e986a200-cdc8-11eb-9b34-7153ab5330d2.png)